compiler/control/OMROptions.cpp

```diff
@@ -684,7 +684,9 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"enableIVTT",                         "O\tenable IV Type Transformation", TR::Options::enableOptimization, IVTypeTransformation, 0, "P"},
    {"enableJCLInline",                    "O\tenable JCL Integer and Long methods inlining", SET_OPTION_BIT(TR_EnableJCLInline), "F"},
    {"enableJITaaSDoLocalCompilesForRemoteCompiles","O\tenable JITaaS to perform local compilations for its remotely compiled methods", SET_OPTION_BIT(TR_EnableJITaaSDoLocalCompilesForRemoteCompiles), "F"},
+   {"enableJITServerFollowRemoteCompileWithLocalCompile","O\tenable JITServer to perform local compilations for its remotely compiled methods", SET_OPTION_BIT(TR_JITServerFollowRemoteCompileWithLocalCompile), "F"},
```

Comment:
Options must be ordered alphabetically, so these new options cannot be placed here. Please move them to their correct alphabetic location.